.github/workflows/test.yml

```diff
@@ -64,6 +64,7 @@ jobs:
             --storybook-build-dir storybook-static \
             --exit-zero-on-changes \
             --exit-once-uploaded \
+            --auto-accept-changes '{main,next}' \
```

Comment:
Not totally super sure this is the right syntax for matching 2 branches 😬 

https://www.chromatic.com/docs/cli#chromatic-options sez we can use globbing (via picomatch), and the interwebz tells me this is how to glob for a list of items.

Figured we can merge this and I'll monitor builds to see what happens.